docs/framework/install/on-windows-7.md

```diff
@@ -14,21 +13,21 @@ ms.assetid: 85cda1d5-c9a4-4eb5-93e6-4f878de07fd7
 
 # Install the .NET Framework on Windows 7 SP1
 
-You may need the .NET Framework to run an application on Windows 7 SP1. You can install the latest supported versions using the following instructions. The instructions depend on which version of the .NET Framework you need.
+You may need the .NET Framework to run an app on Windows 7 SP1. You can install the latest supported versions using the following instructions. The instructions depend on which version of the .NET Framework you need.
 
-## .NET Framework 4.6.2
+## .NET Framework 4.7
 
-You can install the [.NET Framework 4.6.2](https://go.microsoft.com/fwlink/?linkid=845529&source=dotnetdocs) or later on Windows 7.
+You can install the [.NET Framework 4.7](https://www.microsoft.com/net/download/framework) or later on Windows 7.
```

Comment:
i think we should just update the fwlink here. I'll ping @richlander 

also this sentence seems incorrect to me even in the older version. also what about the other versions. you only see the links to others when you get to the dot.net link.